File: cs/src/ConstraintSolver/CNSAT/Types/DecisionLevel.cs
using System;
using System.Collections.Generic;

namespace Alica.Reasoner.CNSAT
{
	public class DecisionLevel
	{
		public int Level {get; set;}
		
		//public double[] Seed {get; set;}
		
		public DecisionLevel (int level)
		{
			this.Level = level;
		}
	}
}

File: java/src/main/java/Reasoner/CNSAT/DecisionLevel.java
//
// Translated by CS2J (http://www.cs2j.com): 12.11.2016 09:40:46
//

package Reasoner.CNSAT;


public class DecisionLevel   
{
    private int Level;
    public int getLevel() {
        return Level;
    }

    public void setLevel(int value) {
        Level = value;
    }

    //public double[] Seed {get; set;}
    public DecisionLevel(int level) throws Exception {
        this.setLevel(level);
    }

}


